### ISSUE TITLE
Mention auto_diag => 1 is incompatible with strict => 1

### ISSUE THREAD

**Issue by jidanni** [makamaka/Text-CSV#40]
On the man page at
```
       strict

        my $csv = Text::CSV->new ({ strict => 1 });
                $csv->strict (0);
        my $f = $csv->strict;

       If this attribute is set to 1, any row that parses to a different number of fields than the previous row will cause the parser to
       throw error 2014.
```
warn that adding it to the example.
```
my $csv = Text::CSV->new( { binary => 1, auto_diag => 1, strict => 1 } );
```
will cause
```
CSV_PP ERROR: 2014 - ENF - Inconsistent number of fields @ rec 236 pos 1
```
at the end of all files, as the parser attempts to get a line beyond the end of the file.
libtext-csv-perl:
  Installed: 1.99-1

**Comment by charsbar** [comment]
Thanks. Fixed the issue and shipped 2.00.

**Comment by ConradHughes** [comment]
Are you sure this is fixed?  With `test.csv` containing anything reasonable (from just the letter 'A' to a few lines of data),
```
use Text::CSV 2.00 'csv';

my $z = csv(in => 'test.csv', strict => 1);
```
.. still seems to trigger error 2014 on the last line plus 1 for me.
